.github/workflows/build-packages.yml

```diff
@@ -21,10 +21,13 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
+      - name: Install VSCE tool
```

Comment:
How much time does this add to the build?

Reply:
Looks like it adds a about a minute. It's not the long pole on the overall build (spread across acquisition and the publish operation itself). It isn't the long pole by any stretch so probably OK.